docs/images/workflow.diag

```diff
@@ -0,0 +1,39 @@
+blockdiag {
```

Comment:
This could be rendered by https://kroki.hzdr.de/

Reply:
I created the diagram with Kroki and put the image URL into the CONTRIBUTING guide: commit f41f927aeb1091487c9200d3784eef946122ae91.

Reply:
@christianhueserhzdr Is it possible to set a white background for the image? 
Since I'm using a dark theme for GitHub, the arrows between the stage were hard to see.

Reply:
This option can only be used for PNG diagrams, so your query string should start with `https://kroki.hzdr.de/blockdiag/png/...?no-transparency=true`.

Nevertheless, at least for me that didn't change anything. :man_shrugging:. It could also be a bug with Kroki. For me it didn't work for both `kroki.io` and our own instance.